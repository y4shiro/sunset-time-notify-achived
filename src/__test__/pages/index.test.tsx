import React from 'react';
import Home from 'src/pages';
import { render } from '../utils';

test('ホームページ', () => {
  const { container } = render(<Home />);
  expect(container).toMatchSnapshot();
});
