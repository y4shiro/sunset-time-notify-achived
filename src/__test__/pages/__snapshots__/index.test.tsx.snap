// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ホームページ 1`] = `
<div>
  <div
    class="css-1ml9zdm"
  >
    <h2
      class="chakra-heading css-gow9n2"
    >
      Sunset Time Notify
    </h2>
    <div
      class="css-17xejub"
    />
    <div
      class="chakra-stack css-1725c0t"
    >
      <div
        class="css-0"
      >
        <p
          class="chakra-text css-0"
        >
          About
        </p>
      </div>
      <div
        class="css-0"
      >
        <p
          class="chakra-text css-0"
        >
          Contact
        </p>
      </div>
    </div>
  </div>
  <main
    class="chakra-container css-1t9qy6v"
  >
    <div
      class="css-1i5b0yu"
    >
      <div
        class="css-n75q0x"
      >
        <img
          class="chakra-image css-11bwpil"
          src="/Map.png"
        />
      </div>
      <div
        class="css-19rkmrw"
      >
        <p
          class="chakra-text css-0"
        >
          日の入の時刻など表示
        </p>
      </div>
    </div>
  </main>
  <div
    class="css-1sv1sar"
  >
    <h2
      class="chakra-heading css-fqf5a4"
    >
      © 2021 y4shiro.
    </h2>
  </div>
</div>
`;
